app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list[dict], cafe: str) -> None | str:
```

Comment:
According to the checklist, you should use parentheses when importing multiple classes/modules. Please consider updating your import statement to follow the recommended style.